### PR TITLE
Validator roll-up

### PR DESCRIPTION
**Pull request by honeybadgerdontcare**
- cl/228469373 Use reserved instead of extensions
- cl/228597059 Revision bump for #20199
- cl/228775442 Revision bump for #19540
- cl/228789212 Add "actions" as a type identifier and related rules.
- cl/228801852 boilerplate no longer has alternative and does not need …

### CODE REVIEW

File: extensions/amp-image-slider/validator-amp-image-slider.protoascii [hunk 1]
```diff
@@ -26,7 +26,6 @@ tags: {  # amp-image-slider
 }
 tags: {  # <amp-image-slider>
   html_format: AMP
-  html_format: EXPERIMENTAL
   tag_name: "AMP-IMAGE-SLIDER"
```
Comment (twifkak) [inline extensions/amp-image-slider/validator-amp-image-slider.protoascii]: Just double-checking this removal is correct.

File: validator/engine/validator.js [hunk 2]
```diff
@@ -4753,9 +4754,9 @@ class ParsedValidatorRules {
           if (validationResult.typeIdentifier.indexOf(identifier) === -1) {
             validationResult.typeIdentifier.push(identifier);
           }
-          // The type identifier "transformed" is not considered mandatory
-          // unlike other type identifiers.
-          if (identifier != 'transformed') {
+          // The type identifier "actions" and "transformed" are not considered
+          // mandatory unlike other type identifiers.
+          if (identifier != 'actions' && identifier != 'transformed') {
```
Comment (twifkak) [inline validator/engine/validator.js]: In the future, should this be a list of those that are mandatory instead?

File: extensions/amp-image-slider/validator-amp-image-slider.protoascii (hunk 1 above)
Comment (twifkak) [reply extensions/amp-image-slider/validator-amp-image-slider.protoascii]: Nevermind, I'll leave the comments on the original CL.

**Comment by honeybadgerdontcare** [comment]
On Fri, Jan 11, 2019 at 11:12 AM Devin Mullins <notifications@github.com>
wrote:

> *@twifkak* approved this pull request.
>
> Just some comments on the original CLs -- the rollup LGTM.
> ------------------------------
>
> In extensions/amp-image-slider/validator-amp-image-slider.protoascii
> <https://github.com/ampproject/amphtml/pull/20274#discussion_r247223655>:
>
> > @@ -26,7 +26,6 @@ tags: {  # amp-image-slider
>  }
>  tags: {  # <amp-image-slider>
>    html_format: AMP
> -  html_format: EXPERIMENTAL
>    tag_name: "AMP-IMAGE-SLIDER"
>
> Just double-checking this removal is correct.
>

This is correct.



> ------------------------------
>
> In validator/engine/validator.js
> <https://github.com/ampproject/amphtml/pull/20274#discussion_r247224149>:
>
> > @@ -4753,9 +4754,9 @@ class ParsedValidatorRules {
>            if (validationResult.typeIdentifier.indexOf(identifier) === -1) {
>              validationResult.typeIdentifier.push(identifier);
>            }
> -          // The type identifier "transformed" is not considered mandatory
> -          // unlike other type identifiers.
> -          if (identifier != 'transformed') {
> +          // The type identifier "actions" and "transformed" are not considered
> +          // mandatory unlike other type identifiers.
> +          if (identifier != 'actions' && identifier != 'transformed') {
>
> In the future, should this be a list of those that are mandatory instead?
>

Probably, but the actions check may be temporary for now.

**Comment by honeybadgerdontcare** [comment]
close/reopen to restart travis

**Pull request merged**